### DISCUSSION TITLE
CallPeaks on multiple fragment files

### DISCUSSION THREAD

**Discussion by jzhou88** [stuart-lab/signac#882] [Q&A] [answered]
Hi Signac People,

I have multiple fragment files and would like to identify a common set of peaks shared by them.
What I'm doing right now is calling CallPeaks only once but with all the fragment files as the input.
I'm wondering if the resulting peaks are just the simple union of the output produced by running CallPeaks on each fragment file.
Or, are the resulting peaks already a common set of peaks shared by all the fragment files?
In the latter, I think I don't need to run GenomicRanges::reduce on these peaks, right?
Thanks in advance for your help.

Best,
J

**Comment by timoast** [comment, marked as answer]
When you supply multiple fragment files, they are all given to a single MACS2 `callpeak` command, which pools the information together. 

> I think I don't need to run GenomicRanges::reduce on these peaks, right?

No, you don't need to run `GenomicRanges::reduce()`

**Reply by jzhou88** [reply]
Got it. Thanks.

BTW, I didn't find the way to compute mitochondria ratio using Signac.
I'm thinking of using the following code:

pbmc[["percent.mt"]] <- PercentageFeatureSet(pbmc, pattern = "^chrM")

But, I'm not sure if this is correct.
If possible, would you mind showing me how to do this correctly?
Thanks in advance.

Best,
J
